src/NServiceBus.Gateway.AcceptanceTests/When_listening_on_wildcard_uri.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace NServiceBus.AcceptanceTests.Gateway
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Config;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_listening_on_wildcard_uri : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void should_throw_exception_if_wildcard_channel_is_used_for_replies()
+        {
+            Assert.That(async () =>
+            {
+                await Scenario.Define<ScenarioContext>()
+                    .WithEndpoint<EndpointWithWildCardUriAsDefault>(e => e.When(b => Task.FromResult(0)))
+                    .Run();
+            }, Throws.Exception.InnerException.InnerException.With.Message.Contains("An extra channel with a fully qualified non-wildcard url must be configured as default for Reply routing."));
```

Comment:
Just noting that when merging to develop `InnerException.InnerException` needs to be removed due to changes in the att framework